src/validateIAPToken.ts

```diff
@@ -28,7 +28,7 @@ export async function getPubKey(
 
 export async function validateIAPToken(
   iapToken: string,
-  expectedAudience: string,
+  expectedAudiences: string | string[],
```

Comment:
2 notes:
- this is a breaking change, so please add the "BREAKING CHANGE: ..." mention in the commit for this change;
- I would simplify the signature to be `string[]` (and migration for existing users would be to just enclose their current expected audience into brackets).

I believe it would simplify the logic down the road.

```suggestion
  expectedAudiences: string[],
```

Reply:
I wanted to avoid breaking change, is it breaking change because of changing the name of the parameter?

Reply:
Oh right 🙈 I somehow misread this as an object parameter (e.g. `function myFn({ param1: string, param2: string})`)

So you're right, it wouldn't be a breaking change by just updating the name of the parameter, my bad.

However, my proposed change of using just `string[]` as a type (as opposed to extending it like you initially proposed) would be one.